controller/orphan_controller.go

```diff
@@ -596,7 +596,7 @@ func (oc *OrphanController) updateDataCleanableCondition(orphan *longhorn.Orphan
 		orphan.Status.Conditions = types.SetCondition(orphan.Status.Conditions, longhorn.OrphanConditionTypeDataCleanable, status, reason, "")
 	}()
 
-	isUnavailable, err := oc.ds.IsNodeDownOrDeletedOrMissingManager(orphan.Spec.NodeID)
+	isUnavailable, err := oc.ds.IsNodeDownOrMissingManager(orphan.Spec.NodeID)
```

Comment:
When the node is deleted, this function will error out. Then the below `reason = longhorn.OrphanConditionTypeDataCleanableReasonNodeDeleted` is unreachable.

Actually, the change in line 613 should be enough.